### PR TITLE
Bug 1384139 - PEP8! PEP8 everywhere! 

### PR DESCRIPTION
**Pull request by adngdb**
Well as of the current state, there are only 277 errors left... :grin: 

I think we want to do this step by step. This is the first one. It only applies changes fixed by [autopep8](https://github.com/hhatto/autopep8), excluding long lines splitting, cause I don't like how it handles that. I am probably just going to go through each file that has errors and will fix them manually, unless I can find another library that does a better job. Anyway, that will come in follow-up pull requests, to make the scope of each smaller and thus easier to review and merge. 

I want to document the fact that we use PEP8 now though before we merge this. Also, at some point we should have our CI lint files, but not before the code base has 0 errors.

### CODE REVIEW

**Comment by jotes** [comment]
:tada: :+1: :hugs: :bowing_man:

**Comment by mathjazz** [comment]
Woohoo!

(I thought the changeset will be bigger!)

**Comment by adngdb** [comment]
As I mentioned, this removes only a handful of PEP8 errors. If you run ``pylama`` in this branch, you still get about 277 errors. However, I don't want to offer you folks a 2k lines changeset, so I'm going to do this in small chunks. Expect many related PRs to come in the future!

**Comment by adngdb** [comment]
@mathjazz I've added a `CONTRIBUTING.rst` file, copied over from [socorro](https://github.com/mozilla-services/socorro), that contains some style guide documentation. There might be things that I am not aware of and that don't apply here. Please let me know what you think about it!

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+Python code should follow PEP-8.
+
+Max line length is 100 characters.
+
+4-space indentation.
+
+To run the linter, do::
+
+  $ pylama pontoon
```
Comment (jotes) [inline CONTRIBUTING.rst]: `pylama` should be enough, all default directories should be specified in `pylama.ini`

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+
+If you hit issues, use ``# noqa`` to make the linter ignore that error. Note that in most cases,
+it is better to fix the issues than ignoring them.
+
+
+Javascript code conventions
+===========================
+
+2-space indentation.
```
Comment (jotes) [inline CONTRIBUTING.rst]: An idea for the next PR: `.editorconfig`

File: CONTRIBUTING.rst (hunk 1 above)
Comment (mathjazz) [inline CONTRIBUTING.rst]: It doesn't work for me (nothing happens or maybe it takes forever), `pylama pontoon` works.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+
+Git conventions
+===============
+
+First line is a summary of the commit. It should start with one of the following::
+
+  Fixes bug XXXXXXX
```
Comment (mathjazz) [inline CONTRIBUTING.rst]: Let's use the [imperative](https://chris.beams.io/posts/git-commit/#seven-rules) mood (Fix bug). Also closes the bug.

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+  Bug XXXXXXX
+
+
+The first, when it lands, will cause the bug to be closed. The second one does not.
+
+After that, the commit should explain *why* the changes are being made and any
+notes that future readers should know for context or be aware of.
+
+
```
Comment (mathjazz) [inline CONTRIBUTING.rst]: Add: "We follow [The seven rules of a great Git commit message](https://chris.beams.io/posts/git-commit/#seven-rules)." (or list them)

File: CONTRIBUTING.rst (hunk 1 above)
Comment (jotes) [inline CONTRIBUTING.rst]: Works almost everytime for me -> maybe there's an unexcluded directory in pylama config and that causes the problem.

**Comment by adngdb** [comment]
I didn't notice this was failing travis. We shouldn't do that. I'm going to work around it. My plan is to ignore the most common errors for now (``E501 line too long`` and ``E128`` continuation line under-indented`` for the moment) and fix the others in this PR.

**Comment by mathjazz** [comment]
Is there anything that remains to be done here? Looks ready to merge!

**Comment by adngdb** [comment]
Yes, I was waiting to see it go green. :) Thanks for the review @jotes and @mathjazz!

**Pull request merged**